accounting.yaml

```diff
@@ -0,0 +1,105 @@
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: accounting-mysql-rc
+  labels:
+    app: accounting
+    tier: accounting-mysql
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: accounting-mysql-rc
+        app: accounting
+        tier: accounting-mysql
+    spec:
+      containers:
+        - name: accounting-mysql
+          image: mysql:5.6
+          ports:
+           - containerPort: 3306
+             name: mysql
+          env:
+           - name: MYSQL_ROOT_PASSWORD
+             value: PutYourPassHere
+           - name: MYSQL_DATABASE
+             value: apel_rest
+           - name: MYSQL_USER
+             value: apel
+           - name: MYSQL_PASSWORD
+             value: PutYourPassHere
+          volumeMounts:
+           - name: accounting-mysql
+             mountPath: /var/lib/mysql
+      volumes:
+        - name: accounting-mysql
+          cinder:
+            volumeID: 44de1b82-2642-4753-915a-54d3d6b54fea
+            fsType: ext4 
+---
+mysql:
+apiVersion: v1
+kind: Service
+metadata:
+  name: accounting-mysql
+  labels:
+    app: accounting
+spec:
+  clusterIP: 10.254.10.21
```

Comment:
This IP appears a lot - is it possible to have it as a single config option?

It should also probably not be a specific IP address in the repo.